.github/workflows/yarn_docker_image.yml

```diff
@@ -70,6 +70,12 @@ jobs:
                 if: ${{ inputs.poeditor_translations_loc != '' }}
                 with:
                     python-version: '3.x'
+                    cache: 'pip'
+
+            -   name: Install required python module using pip
+                if: ${{ inputs.poeditor_translations_loc != '' }}
+                run: |
+                    pip install requests
```

Comment:
Wouldn't it be better to have a requirements.txt. Then you wouldn't need to update this action if you'd ever change the packages needed?